parsing/src/common.ts

```diff
@@ -23,5 +51,19 @@ function isFicsmas(displayName: string): boolean {
         displayName.includes("Fireworks");
 }
 
+function getRecipeName(name: string): string {
+    return name.replace("Build_", "").replace(/_C$/, "");
+}
+
+function getPartName(name: string): string {
+    name = name.replace('Desc_', '').replace(/_C$/, '');
+    return name;
+}
+
+function getFriendlyName(name: string): string {
+    // Remove any text within brackets, including the brackets themselves
+    return name.replace(/\s*\(.*?\)/g, '');
+}
+
 // Export various items for use
-export {blacklist,isFluid,isFicsmas}
\ No newline at end of file
+export {blacklist,whitelist,isFluid,isFicsmas,getRecipeName,getPartName,getFriendlyName}
```

Comment:
fyi you'll get bonked by the linter for this one
```suggestion
export {
  blacklist,
  whitelist,
  isFluid,
  isFicsmas,
  getRecipeName,
  getPartName,
  getFriendlyName
}
```